### PR TITLE
Added program for infix to postfix conversion using stack

### PR DESCRIPTION
**Pull request by Harinath-B**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by appgurueu — CHANGES_REQUESTED** [review]
1. Fix your formatting. Please also use syntactical sugar like `++` or `--`.
2. **No side effects.** Remove calls to `console.log`.

**Review by appgurueu — COMMENTED** [review]
Please add some more tests, with more descriptive test names

File: Data-Structures/Stack/ToPostfix.js [hunk 1]
```diff
@@ -49,38 +49,32 @@ const isAlNum = (c) => {
 }
 
 const priority = (op) => {
+  op = String(op)
```
Comment (appgurueu) [inline Data-Structures/Stack/ToPostfix.js:52]: This is unnecessary.

**Review by appgurueu — CHANGES_REQUESTED** [review]
Still: Please add more complex tests. Testing one simple expression and the edge case of an empty expression is good but not enough.

File: Data-Structures/Stack/ToPostfix.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ class Stack {
   // Returns the last element without removing it
   last () {
     if (this.top !== -1) {
-      return this.stack[this.stack.length]
+      return this.stack[this.length()]
```
Comment (appgurueu) [inline Data-Structures/Stack/ToPostfix.js:41]: Does this work with 0-indexing?
